explainer.md

```diff
@@ -0,0 +1,147 @@
+# Open Screen Protocol Explainer
+
+The Open Screen Protocol is a standard way for two agents to discover each other
+on the local network, create a secure network connection, and allow an
+application on one agent to render media on the other.
+
+This capability allows users to discover media on one device (such as a laptop
+or smartphone) and play it on another device (such as a smart TV, set top box,
+or connected speaker).   We call this scenario "second screen" functionality.
+
+## Why should users care?
+
+Currently there are many solutions that allow media accessed on one device to be
+played on a different device.  Examples include [Google
+Cast](https://developers.google.com/cast/), [Apple
+AirPlay](https://www.apple.com/airplay/), and [Spotify
+Connect](https://www.spotify.com/us/connect/).
+
+However, the application, browser and/or OS largely determine what devices work
+together for media playback.  Users can't be easily guaranteed that the content
+that they watch and the browser they choose will be able to interface with the
+smart TV or speaker that they have or want.
+
+## Why should Web developers care?
+
+Web developers who want to integrate second screen functionality into their
+applications need to write different code for each browser to interface with the
+devices supported by that browser.  This discourages wide adoption of remote
+playback functionality.  For applications that do add second screen support,
+they tend to pick one device type versus integrating multiple device-specific
+SDKs into their application.
```

Comment:
And this is bad because it limits the types of devices compatible with certain content?

Reply:
Yes.